pilot/pkg/networking/core/v1alpha3/networkfilter.go

```diff
@@ -116,16 +101,63 @@ func buildDeprecatedTCPProxyFilter(clusterName string, addr string) (*listener.F
 	return tcpFilter, nil
 }
 
-// buildOutboundNetworkFilters generates TCP proxy network filter for outbound connections. In addition, it generates
-// protocol specific filters (e.g., Mongo filter)
-// this function constructs deprecated_v1 routes, until the filter chain match is ready
-func buildOutboundNetworkFilters(env *model.Environment, node *model.Proxy, clusterName string,
-	deprecatedTCPFilterMatchAddress string, port *model.Port) []listener.Filter {
+// buildStatPrefix builds a stat prefix by concatenating the given route
+// destinations and port.
+func buildStatPrefix(routes []*networking.RouteDestination, port int, push *model.PushContext) string {
```

Comment:
there is no max length for stats prefixes?

Reply:
Addressed via c164e2d.